codex_of_the_damned/templates/archetypes/bibliodeque/419-operation.html

```diff
@@ -0,0 +1,48 @@
+{% extends "archetypes/_layout.html" %}
+{% import "archetypes/_layout-erratum.html" as erratum with context %}
+{% import "archetypes/bibliodeque/419-operation.json" as deck_json %}
+
+{% block meta %}
+<meta property="decklist" content='{{ deck_json | tojson }}' />
+{% endblock %}
+
+{% block archetype_name %}
+<h1><img src="{{ url_for('static', filename='img/icons/clan-osebo.svg') }}"> 419 Operation</h1>
+{% endblock %}
+
+{% block archetype %}
+{% trans trimmed
+	operation=card("419 Operation"),
+    hidetheheart=card("Hide the Heart"),
+    kovalcik_deck=link("/deck-search", _("Matus Kovalcik's deck"), twda_id="2014sncqks")
```

Comment:
link is broken

Reply:
Tried manually today on release website and the link is not broken
https://codex-of-the-damned.org/fr/deck-search.html?twda_id=2014sncqks

Reply:
Does not work in local environment but link seems fine